src/Hintman/Suggest.hs

```diff
@@ -0,0 +1,40 @@
+{-# LANGUAGE NamedFieldPuns #-}
+
+module Hintman.Suggest
+       ( getTrailLines
+       , toLines
+       ) where
+
+import Data.IntMap.Strict (IntMap, traverseWithKey)
+
+type Lines = IntMap Line
+
+data LineState = Plain | Trailing
+    deriving (Eq, Show)
+
+data Line = Line
+    { lName  :: Text
```

Comment:
Hmm, is it FilePath? Why do we need to store it for each line?
I guess it's better to find trailing spaces for each file separately. What do you think?

Reply:
> What do you think about it?

I think it is cool suggestion! :)

Reply:
> What do you think?

`lName` is lagacy code))